python/assign.py

```
@@ -1,5 +1,11 @@
 import firebase_admin
 import pulp
+import smtplib
+import os
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.mime.base import MIMEBase
```

Ruff (0.11.9)

`email.mime.base.MIMEBase` imported but unused

Remove unused import: `email.mime.base.MIMEBase`

(F401)